dlc_developer_config.toml

```diff
@@ -62,7 +66,7 @@ use_scheduler = false
 # Standard Framework DLC buildspecs
 dlc-pr-autogluon = ""
 dlc-pr-mxnet = ""
-dlc-pr-pytorch = ""
+dlc-pr-pytorch = "pytorch/buildspec-sm-lite-1-13.yml"
```

Comment:
This PR just adds placeholder dockerfiles and buildspec files for the sm lite DLCs. I've just copy-pasted the PT 1.12 dockerfiles with minimal changes, and I've refrained from adding a "prod" buildspec file for sm lite DLCs. Since you've written actual dockerfiles and buildspec files already, I'd like to go through your PR to add this stuff.

There isn't anything wrong with using the framework version numbers on the buildspec. It's just that this buildspec is not one that is going to be used by default on any of the pipelines, and is meant to exist just as a reference, or deleted when your PR is merged.